src/elm/View/Pin.elm

```diff
@@ -0,0 +1,109 @@
+module View.Pin exposing (..)
+
+import Html exposing (..)
+import Html.Attributes exposing (..)
```

Comment:
elm-review will probably complain about this `exposing (..)`

Reply:
Now we haven't this rule enabled. Just added it to use in the future. I tried it on the whole project and it worked well since we don't expose everything anywhere else.

Reply:
oh I see! good one @amiskov